src/lib/ext-monitor.js

```diff
@@ -1,6 +1,10 @@
 import { getActivityLogPageURL } from './ext-listen.js';
 
 export default class ExtensionMonitor {
+  /**
+   * @type {Array.<{logs: Array<Object>, fileName: String}>}
+   */
+  loadedLogs = [];
```

Comment:
Similar to my other comment, if the activitylog.html page opened for a loaded log file is only being used to review those logs, we may not need both `loadedLogs` and `logs` (eg. that instance of the activitylog.html page is not meant to subscribe to receive realtime logs at all).

Reply:
@rpl here the `loadedLogs` array is in `ext-monitor.js` (background) where I am storing the loaded logs along with the filename. And when I am opening a new tab with parameter like: `?page=loadLog&file=fileName.json`, the activitylog.html getting the loaded logs from the background and [renders the logs in the table view](https://github.com/mozilla/extension-activity-monitor/pull/28/files#diff-c00fd244576d6e063a30656f921d4db6R182)

I am storing the loaded logs in the background since, if a user mistakenly closes the tab / reloads the tab, he doesn't lose the loaded logs. If he mistakenly closes the tab, he will still be able to see the loaded logs adding the URL parameters: `?page=loadLog&file=fileName.json`.

Reply:
@atiqueahmedziad ah, I see. Thanks for pointing it out!

Wouldn't be reasonable to store the logs in a map keyed by the filename string?

Reply:
Yeah, it's reasonsable :-) Thanks.
I have updated the PR storing the logs and file name in a Map.